libs/api/books/src/lib/reading-list.controller.ts

```diff
@@ -16,8 +16,18 @@ export class ReadingListController {
     return await this.readingList.addBook(item);
   }
 
+  @Delete('/reading-list/:id')
```

Comment:
I'm surprised the API still works with two routes exactly the same.

You don't need to do this.